content/articles/account-users.markdown

```diff
@@ -34,17 +34,17 @@ The member will also have full-admin access to the account, including the abilit
 
     ![Select the members tab](/files/add-member-account-link.jpg)
 
-1. Click the <label>Add a new member</label> link.
+1. Click the <label>Invite new user</label> link.
 
-    ![Add a new member](/files/add-member-link-members-screen.jpg)
+    ![Add a new member](/files/add-member-link-members-screen.png)
 
-1. Enter the email address of a __registered__ DNSimple customer. It is not necessary that this person have an active subscription, just that they have already signed up for the service.
+1. Enter the email address of of the user that you wish to invite. It doesn't matter if they have a DNSimple account or not. They will receive an email inviting them to join the account. 
 
     ![Add a new member form](/files/add-member-screen.png)
```

Comment:
Can I ask you to use a screenshot with a team name that is generic, and not specific to RubyGems? 😉